src/core/store.ts

```diff
@@ -26,7 +27,7 @@ import extractServiceBaseUrls from "./utils/reduxMiddleware/extractServiceBaseUr
 const persistConfig = {
   key: "dpl-react",
   storage,
-  blacklist: ["text", "url", "modal", "config"]
+  blacklist: ["text", "url", "modal", "config", "blockedModal"]
```

Comment:
Are you shure you want to blacklist it?
I am pretty shure that with all pageloads, eg. creating a search would reset the state unless you persist it.